test/utils.py

```diff
@@ -0,0 +1,51 @@
+from collections.abc import Sequence
+from dataclasses import dataclass
+
+import numpy as np
+from gymnasium import spaces
+
+from tianshou.data.collector import CollectStats
+
+
+@dataclass
+class SpaceInfo:
+    action_shape: int | Sequence[int]
+    state_shape: int | Sequence[int]
+    action_dim: int
+    state_dim: int
+    min_action: float
+    max_action: float
```

Comment:
I suggest that you add a constructor method here called `from_space` or something like that. Then you can delete the `get_space_info` function

Reply:
Good point. I fixed it (and will push asap).

I saw similar things done in the code-base (e.g., in data.stats.py).